engine/src/stack.cpp

```diff
@@ -1379,7 +1373,10 @@ Boolean MCStack::del(bool p_check_flag)
 {
     if (!isdeletable(true))
 	   return False;
-	
+    
+    MCscreen->ungrabpointer();
+    MCdispatcher->removemenu();
```

Comment:
I'm not positive these are reasonable... I suspect I need to add a way to ungrab if it's the stack window that is grabbed and the same with removemenu... on the other hand while a popup grabbed are we really deleting stacks?